src/injected/main-window-initializer.ts

```diff
@@ -206,9 +204,12 @@ export class MainWindowInitializer extends WindowInitializer {
         await Promise.all(asyncInitializationSteps);
     }
 
-    @autobind
     protected dispose(): void {
-        super.dispose();
+        // Using .bind(this) instead of an arrow function because ES5 only supports accessing methods (not properties) of super
+        // When we move to target ES6, this can be simplified to super.dispose()
+        // See https://github.com/microsoft/TypeScript/issues/338
+        super.dispose.bind(this)();
```

Comment:
> super.dispose.bind(this)(); [](start = 7, length = 28)

when u bind, super.dispose does it also bind this.dispose?

Reply:
No, good catch. The correct place to do this is actually at the callsite of the child. Fixed this.